requirements/pip.txt

```diff
@@ -92,9 +92,10 @@ dj-pagination==2.5.0
 # Version comparison stuff
 packaging==21.2
 
-# django-cors-middleware==1.5.0 fails with
+# django-cors-middleware was replaced by django-cors-headers
+# django-cors-headers==3.10.0 is failing with:
```

Comment:
Should we not fix the error if we're looking to upgrade this package? This feels like a sideways move, but not really what we need to do to move forward?

Reply:
I don't think we can fix this issue by ourselves. I didn't research too much yet, but I found https://github.com/adamchainz/django-cors-headers/issues/558 that it's related to a Django bug which is solved in 3.1.x

Reply:
@humitos I believe we're on a newer Django now, so is this fixed?

Reply:
Let's see. I've rebased the branch and push. Tests should pass now if it's fixed 😄

Reply:
It seems it's not fixed yet... Tests are still failing, :/